.github/workflows/codeql-analysis.yml

```
@@ -36,14 +36,26 @@ jobs:
       with:
         languages: ${{ matrix.language }}
         debug: false
-        config-file: ./.github/codeql/codeql-config.yml              
-    
-    - name: Build
+        config-file: ./.github/codeql/codeql-config.yml
+
+    - name: Build JavaScript
+      uses: github/codeql-action/autobuild@v2
+      if: ${{ matrix.language != 'go' }}
```

```suggestion
      if: ${{ matrix.language == 'javascript' }}
```

This would seem safer in case we ever add any more languages.

---

Indeed. I updated it.